.github/workflows/presubmit.yaml

```diff
@@ -70,7 +70,7 @@ jobs:
         name: testlogs
         path: bazel-testlogs
   e2e-tests-orchestration:
-    runs-on: ubuntu-22.04 
+    runs-on: ubuntu-22.04-4core
```

Comment:
this costs money, it's ok to continue using standard runner here.

Reply:
After applying this PR, it hit the storage limitation. I suspect storage usage for building each stage of docker image matters, but haven't investigated yet.

I compared specification between `ubuntu-22-04` and `ubuntu-22.04-4core` runners and thought changing the runner can solve this issue.
- `ubuntu-22.04`: 4 core / 16GB RAM / 14GB SSD
- `ubuntu-22.04-4core`: 4 core / 16GM RAM / 150GB SSD

But I'm not saying that we have no alternative options for that. If using this runner is not free, I'll seek for the other apporach for validating this runner is the only option that we can pick.

Reply:
Avoided storage needs by removing intermediate containers. Done.